openpype/pipeline/create/context.py

```diff
@@ -1804,10 +1804,10 @@ def _reset_creator_plugins(self):
                 self,
                 self.headless
             )
+            creators[creator_identifier] = creator
```

Comment:
I don't think this is ok.

Reply:
But how is the `rig` family going to load in?

Reply:
Actually hold on. Might have a solution